TenureInformationApi/V1/Controllers/TenureInformationController.cs

```diff
@@ -62,7 +62,9 @@ public async Task<IActionResult> GetByID([FromRoute] TenureQueryRequest query)
             var result = await _getByIdUseCase.Execute(query).ConfigureAwait(false);
             if (result == null) return NotFound(query.Id);
 
-            HttpContext.Response.Headers.Add(HeaderConstants.ETag, EntityTagHeaderValue.Parse($"\"{result.VersionNumber.Value}\"").Tag);
+            int versionNumber = (result.VersionNumber != null) ? result.VersionNumber.Value : 0;
```

Comment:
That's not right and will fail when the the FE tries to update the record using this value.
If there is no version number value then we return no value in the ETag header- though I'm not sure whether this would mean null/an empty string or nothing between 2 quotes.
@amidomichael Any ideas on the 2 options above?

The person api has this same issue.

Reply:
Nothing between two quotes works on Person API.